### DISCUSSION TITLE
Library for unit handling

### DISCUSSION THREAD

**Discussion by dvd101x** [mgreminger/EngineeringPaper.xyz#203] [Ideas]
Hi,

I was reading into the docs that **EngineeringPaper.xyz** is using *mathjs* for units handling, which it does very well. Nonetheless they have the intention to use *UnitMath* for units.

Since *mathjs* is very broad and does many other things besides units, you might consider using *UnitMath*.

https://github.com/ericman314/UnitMath
https://github.com/josdejong/mathjs/issues/2081#issuecomment-768379768

**Comment by mgreminger** [comment]
In addition to the conversion factors, the bignumber arbirtrary precision capability of mathjs is used to convert the values to and from the SI units used for the calculations to reduce precision loss.  As far as I can tell, UnitMath uses the javascript number type so there would be precision loss in switching to UnitMath.

**Reply by dvd101x** [reply]
My bad, I didn't notice the use of bignumber, thanks for the response and consideration.
